### PR TITLE
LPS-178990 | Poshi Automation for Bug | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# LPS-178990 | Poshi Automation for Bug

Hi! :)
This pr is corresponding to this ticket: [LPS-143321](https://issues.liferay.com/browse/LPS-143321) → [LPS-178990](https://issues.liferay.com/browse/LPS-178990)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 1]
```diff
@@ -1578,6 +1578,61 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-143321. After clicking on discard draft the page continues to run."
+	@priority = 5
+	test ViewFragmentsAfterDiscardDraft {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:1583]: Move the case to ContentPagesWithExperiencePersonalization.testcase
Rename it to DiscardContentPageDraft

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 2]
```diff
@@ -1578,6 +1578,61 @@ definition {
...
+		task ("Given a page with published fragments") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:1584]: Given a page designer has a content page with draft and experience

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 3]
```diff
@@ -1578,6 +1578,61 @@ definition {
...
+				menuItem = "Edit",
+				pageName = "Test Content Page Name");
+
+			PageEditor.addExperience(
+				experienceName = "Experience Name",
+				segmentName = "Anyone");
+
+			PageEditor.clickBack();
+
+			PagesAdmin.discardDraft(pageName = "Test Content Page Name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:1614]: When the page designer discards draft via Pages admin

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 4]
```diff
@@ -1578,6 +1578,61 @@ definition {
...
+
+			Alert.viewSuccessMessage();
+		}
+
+		task ("Then the page fragments should not have errors") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:1619]: Then the page editor should be rendered well

**Review by Tim-Cao — COMMENTED** [review]
Hi @rafaellajordao, would you mind refactoring according to comments? Thanks

**Comment by Tim-Cao** [comment]
@rafaellajordao see changes https://github.com/liferay-echo/liferay-portal/pull/12417

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**